### ISSUE TITLE
How to approach assets from the Track & Trace AllThingsTalk Kit with this SDK?

### ISSUE THREAD

**Issue by digiteyezer** [allthingstalk/python-sdk#3]
Dear,

When your track & trace device(ATT kit) is configured on the ATT platform (using Enco as LoRa network), assets names are mentioned as numbers matching the LoRa container ID.
I'm not able to subscribe to the feeds of these assets getting an error in the SDK. 

Kind regards,
Filip

**Comment by vydd** [comment]
Hello,

There was indeed a bug with handling of custom names, which was just fixed. You should be able to install 0.1.8 using pip and make this work.

Now, subscribing to assets whose name is not a valid Python identifier (like 1 or 2) might not be obvious, so a section in documentation was added to fix that, and you can find it here: https://allthingstalk.github.io/python-sdk/#using-invalid-python-identifiers-for-asset-names

Basically, instead of trying to do something like

```python
class Kit(Device):
    1 = BooleanAsset()

@Kit.feed.1
...
```

which will of course fail, you can choose a human readable name for your identifier, and supply the real name when specifying the asset:

```python
class Kit(Device):
    switch = BooleanAsset(name='1')

@Kit.feed.switch
...
```

Hope this works for you.
